usvm-jvm/src/main/kotlin/org/usvm/machine/state/JcState.kt

```diff
@@ -36,13 +39,17 @@ class JcState(
     targets
 ) {
     override fun clone(newConstraints: UPathConstraints<JcType>?): JcState {
-        val clonedConstraints = newConstraints ?: pathConstraints.clone()
+        val newThisOwnership = MutabilityOwnership()
+        val cloneOwnership = MutabilityOwnership()
+        val clonedConstraints = newConstraints?: pathConstraints.clone(newThisOwnership, cloneOwnership)
```

Comment:
After successful fork `this.pathConstraints` will have modified ownership (different from newThisOwnership)